### PR TITLE
feat: Add function to return txt as string

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
Add a function to return the generated `ensight_api_test_assets.txt` as a string.

### CODE REVIEW

**Comment by mariostieriansys** [comment]
@AlejandroFernandezLuces is your idea to include the assets folder in the wheel, so that the assets can then be called in the relevant unit test in PyEnSight?

I like the idea to avoid loading a file in the unit test but just get a string back.

File: codegen/access_assets.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+def ensight_api_test_assests():
+    """Read ensight_api_test_assets.txt and return it as string.
+
+    Returns
+    -------
+    str
+        The file contents as string.
+    """
+    root = os.path.dirname(os.path.dirname(__file__))
+    assets_file_name = os.path.join(root, "assets", "ensight_api_test_assets.txt")
```
Comment (RobPasMue) [inline codegen/access_assets.py:12]: Assume the file qould be in its existing location... or in an assets folder inside stc/ansys/api/pyensight. Whatever you prefer

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: codegen/stub_api.py [hunk 2]
```diff
@@ -758,7 +758,7 @@ def process(self, dirname: str, filename: str) -> None:
         with open(filename, "w") as f:
             f.write(s)
         root = os.path.dirname(os.path.dirname(__file__))
-        assets_file_name = os.path.join(root, "src", "ansys", "api", "pyensight","ensight_api_test_assets.txt")
+        assets_file_name = os.path.join(root,"src", "ansys", "api", "pyensight", "assets", "ensight_api_test_assets.txt")
```
Comment (RobPasMue) [inline codegen/stub_api.py:761]:
```suggestion
        assets_file_name = os.path.join(root, "src", "ansys", "api", "pyensight", "assets", "ensight_api_test_assets.txt")
```

File: src/ansys/api/pyensight/access_assets.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+def ensight_api_test_assests():
+    """Read ensight_api_test_assets.txt and return it as string.
+
+    Returns
+    -------
+    str
+        The file contents as string.
+    """
+    root = os.path.dirname(os.path.dirname(__file__))
+    assets_file_name = os.path.join(root,"src", "ansys", "api", "pyensight", "assets", "ensight_api_test_assets.txt")
```
Comment (RobPasMue) [inline src/ansys/api/pyensight/access_assets.py:12]:
```suggestion
    dir = os.path.dirname(__file__)
    assets_file_name = os.path.join(dir, "assets", "ensight_api_test_assets.txt")
```

**Review by RobPasMue — CHANGES_REQUESTED** [review]
Some changes

**Review by RobPasMue — APPROVED** [review]
As long as the TXT file is being packed as part of the wheel, this LGTM. Please verify before merging @AlejandroFernandezLuces

**Pull request merged**